letsencrypt/tasks/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+
+- name: Install Let's Encrypt
+  apt: name="letsencrypt"
```

Comment:
The package name and command vary by platform, but it's `letsencrypt` on Xenial. I'll make this a variable so it's generic.